rb-monitor.init.upstart

```diff
@@ -0,0 +1,24 @@
+description "rb-monitor service" 
+author      "jjprieto@redborder.com" 
+
+start on filesystem or runlevel [2345]
```

Comment:
Not needed. This is only a base systemd service thinking in a supervise as chef that command the start or stop of the service.